samples/bluetooth/direction_finding_central/README.rst

```diff
@@ -85,6 +85,10 @@ This also means that, for example, when using four GPIOs, the pattern count cann
 
 If the number of switch-sample periods is greater than the number of stored switching patterns, then the radio loops back to the first pattern.
 
+Maximum length of antenna switch pattern is limited by :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
+If required length of the antenna switching pattern is greater than default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` add the Kconfig to a board configuration file.
+For example for :ref:`nRF52833 DK <ug_nrf52>` add :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`=N, where N is required antanna switch pattern length, to :file:`nrf52833dk_nrf52833.conf`
```

Comment:
We are missing articles in this sentence.
```suggestion
If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` add the Kconfig to a board configuration file.
For example for :ref:`nRF52833 DK <ug_nrf52>` add :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`=N, where N is the required antenna switch pattern length, to :file:`nrf52833dk_nrf52833.conf`
```

I also have an alternative formulation below that I think is simpler. (If you prefer.)

```suggestion
The value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` can be set in a board configuration file.
E.g. for :ref:`nRF52833 DK <ug_nrf52>` add :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` with the desired value to :file:`nrf52833dk_nrf52833.conf`
```

Reply:
Corrected in all README.rst